ui/src/common/utils.js

```diff
@@ -208,27 +208,9 @@ export function downloadTextAsFile(
   type = 'application/txt',
   extension = 'txt'
 ) {
+  const date = moment().format('YYYY-MM-DDTHHmmss');
```

Comment:
was this a requirement?

Reply:
I discussed it with Stella as a way of differentiating citation files

Reply:
this is not good, if you use current download date as a name, "Downloads" folder will have duplicates as you download the same stuff and since the name is different for each time, there won't be easy way for user to see which files are duplicates.

Reply:
What do you mean `search` for CiteAll @harunurhan ?

Reply:
I mean we can't set record id for file that is dowloaded via CiteAll, since there are many records. So we could just name it `search`

Reply:
There's not much value for this and we can't avoid having some duplicates. So let's do 'INSPIRE-Cite-record' for single ones and 'INSPIRE-CiteAll' for the rest.

Reply:
So we go without dates right?

Reply:
yes